File: DotNet/CesiumLanguageWriter/Advanced/ICesiumReferenceListValuePropertyWriter.cs
using System.Collections.Generic;

namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a list of references to other properties.
    /// </summary>
    public interface ICesiumReferenceListValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a list of references.
        /// </summary>
        /// <param name="references">The list of references.</param>
        void WriteReferences(IEnumerable<Reference> references);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumReferenceListValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 *  
 A writer that can write a value as a list of references to other properties.
 

 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumReferenceListValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    *  
    Writes the value as a list of references.
    
    

    * @param references The list of references.
    */
    void writeReferences(Iterable<Reference> references);
}